src/stock-home/stock-home-receiving-modal.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Modal, Table, TableHead, TableRow, TableHeader, TableBody, TableCell, TableContainer } from '@carbon/react';
+
+const ReceivingStockModal = ({ open, onClose, receivingStock }) => {
+  const headers = [
+    { key: 'status', header: 'Status' },
+    { key: 'sourceName', header: 'Source' },
+    { key: 'destinationName', header: 'Destination' },
+    { key: 'stockItemName', header: 'Stock Item' },
+    { key: 'stockItemPackagingUOMName', header: 'Unit' },
+    { key: 'quantity', header: 'Quantity' },
+  ];
+
+  return (
+    <Modal
```

Comment:
lets use the stick to using the `openmrs/esm-framework` utilities for modal creation and management